src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -364,6 +358,23 @@ bool mesh::is_multidomain(const Node &n,
 }
 
 
+//-----------------------------------------------------------------------------
+bool
+mesh::verify(const Node &n,
+             Node &info)
+{
+    return mesh::verify_multidomain(n, info) || mesh::verify_unidomain(n, info);
```

Comment:
I think we can check single domain, if that fails check multi. 
We can clear the info for the second case.